<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NatusferaQGIS3DialogDownloading</class>
 <widget class="QDialog" name="NatusferaQGIS3DialogDownloading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>89</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>89</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>89</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Downloading...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="downloading_cancel_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="downloading_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Downloading</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="downloading_progress_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
